package site.dogether.challengegroup.exception;

public class FinishedChallengeGroupException extends RuntimeException {
    public FinishedChallengeGroupException(String message) {
        super(message);
    }
}
